src/nextjs/index.ts

```diff
@@ -125,7 +125,7 @@ export class OttofellerNextjsProject extends NextJsTypeScriptProject {
     }
 
     // ANCHOR Github workflow
-    PullRequestTest.addToProject(this, options)
+    PullRequestTest.addToProject(this, {...options, isLighthouseEnabled: false})
```

Comment:
With such setup we'll never instantiate lighthouse workflow. Is this a mistake, or do we get rid of lighthouse altogether?

Reply:
@quesabe It's not a mistake. @bdoof and I discussed the lighthouse and concluded that after this PR we will make the lighthouse a standalone project as well

Reply:
@bdoof You mentioned the following:
> Let's not touch any Lightouse related stuff in this PR (except the flag rename).
Currently this change does not align and we still with nextjs having lighthouse completely disabled. If that's the intent, then feel free to resolve this conversation. My point was to only draw attention to the state of lighthouse in nextjs.

Reply:
@bdoof You mentioned the following:
> Let's not touch any Lightouse related stuff in this PR (except the flag rename).

Currently this change does not align and we still with nextjs having lighthouse completely disabled. If that's the intent, then feel free to resolve this conversation. My point was to only draw attention to the state of lighthouse in nextjs.